store/xeps.js

```diff
@@ -0,0 +1,74 @@
+const $states = {
+    server: "SERVER",
+};
+
+const $actions = {
+    updateXepsWithDisco: "UPDATE_XEP_WITH_DISCO",
+};
+
+const $getters = {
+    getServerXep: "GET_SERVER_XEP",
+};
+
+const $mutations = {
+    setServerXep: "SET_SERVER_XEP",
+}
+
+export const state = () => ({
```

Comment:
We should probably add comments here describing every XEP/RFC, e.g. 
```js
"XEP-0030": false, // service discovery
```

Reply:
You have an explanation in [`/assets/xeps/server/data.js`](https://github.com/bonfire-xmpp/bonfire/blob/feature/mobile-layout/assets/xeps/server/data.js) locally, and running an online search should help. Sadly, such is the way of the XEP. And I don't feel like rewriting tons of boilerplate comments.